coalib/parsing/CliParsing.py

```diff
@@ -87,3 +87,31 @@ def parse_custom_settings(sections,
                                origin=origin,
                                section_name=key_touple[0],
                                from_cli=True)
+
+
+def check_conflicts(cli_sections, log_printer):
```

Comment:
Yep, just found that this: https://docs.python.org/3/howto/argparse.html#conflicting-options

Basically you add

```
group = parser.add_mutually_exclusive_group()
group.add_argument("-v", "--verbose", action="store_true")
group.add_argument("-q", "--quiet", action="store_true")
```

and this automatically raises "`-v` and `-q` can't be given at the same time". ArgParser is really complete!